data/core/init.lua

```diff
@@ -577,15 +826,32 @@ function core.remove_project_directory(path)
   return false
 end
 
-local function reload_on_user_module_save()
+
+local function whitespace_replacements()
+  local r = renderer.replacements.new()
```

Comment:
I don't think we use `renderer.replacements` anymore.

On this note, the new `drawwhitespace` plugin doesn't allow customizing them; we should change that.

Reply:
Right, this landed there by accident. I am going to remove this function, thank you for reporting the problem.